pkg/shell/base_index.js

```diff
@@ -362,6 +362,14 @@ var phantom_checkpoint = phantom_checkpoint || function () { };
                     perform_jump(child, control);
                     return;
 
+                } else if (control.command == "logout") {
+                    cockpit.logout(true);
+                    return;
```

Comment:
If we do that then it isn't possible to get the reload behavior from inside the frame, which I sort of assumed would be the more common usage. But maybe that's acceptable. WDYT?